scripts/scil_visualize_bundles_mosaic.py

```diff
@@ -10,44 +10,60 @@
 import argparse
 import logging
 import os
+import random
 import shutil
 
-import nibabel as nib
 from fury import actor, window
+import nibabel as nib
+import numpy as np
 
 from PIL import Image
 from PIL import ImageFont
 from PIL import ImageDraw
 
-from scilpy.io.utils import (
-    add_overwrite_arg, assert_inputs_exist, assert_outputs_exist)
+from scilpy.io.image import get_data_as_mask
+from scilpy.io.utils import (add_overwrite_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+from scilpy.utils.filenames import split_name_with_nii
```

Comment:
While we are working this script can you please add 1-2 examples in the docstring that cover options and summarize your PR comment. Thanks